platform/ios/PolyPodApp/PodApi/PolyOut/FS/PolyOut+FS.swift

```diff
@@ -107,7 +107,7 @@ extension PolyOut {
     
     func fileRead(url: String, options: [String: Any], completionHandler: @escaping (Any?, Error?) -> Void) {
         do {
-            guard let filePath = pathFromUrl(url: url) else {
+            guard let filePath = filesystemUrlFromPodUrl(url: url) else {
```

Comment:
More carried over code. I would use uri, and not URL here, since we're talking about resource identifiers for files. Also, I would have to take a look at how `noSuchFile` works, but giving it an URL can be kind of confusing; an URI wouldn't be so confusing, but even so, the kind of error that's been thrown is not exactly noSuchFile, but "no file mapping for this URI", or maybe something more complicated.